README.md

```diff
@@ -20,7 +20,7 @@
 
 | Folder | Version | Changelog | Description |
 | ------ | ------- | --------- | ------- |
-| [/api-demo](./api-demo/) | (local project) |  | Code samples illustrating how to use the **@microosft/tsdoc** parser |
+| [/api-demo](./api-demo/) | (local project) |  | Code samples illustrating how to use the **@microsoft/tsdoc** parser |
```

Comment:
This table is generated actually. I think we need to fix the package.json file as well.